java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -289,10 +289,11 @@ CompletableFuture<Long> bitop(
 
     /**
      * Reads the array of bits representing the string that is held at <code>key</code> based on the
-     * specified <code>subCommands</code>.
+     * specified <code>subCommands</code>. Can be routed to read-only replicas to allow for <code>
+     * BitField</code> behaviour in read-only replicas.
```

Comment:
The phrase is correct and not at the same time.
We do not provide routing option for keyed commands. Should we add it here? Routing is determined automatically by GLIDE and user can do nothing there. A notice is correct, but useless.

Reply:
Should it just be `allows for <code>BitField</code> behaviour in read-only replicas.`?

Reply:
As discussed with @GilboaAWS, we need to copy it from #1528
> This command is routed depending on the the client's `ReadFrom` strategy.

Reply:
Add `{@link ..}` to this setting.

Reply:
Updated. Thanks!